src/docs/background/20-time-based-template/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Time-based template
+slug: time-based-template
+updated: 2019-10-21
+category: background
+ingress:
+  This article introduces the time-based template and what's different compared
+  to the day-based FTW.
+published: true
+---
+
+Time-based template adds support for time-based bookings. By default, template
+uses a transaction process
+[preauth-unit-time-booking](https://github.com/sharetribe/flex-example-processes/tree/master/preauth-unit-time-booking)
+which is a unit-based process and has value `time` in `create-booking` booking
+action.
+
+> **Note:** We have removed `BookingDatesForm` and `EditListingAvailabilityForm`
+> components form the template as they are heavily related to the day-based
+> process. If you have started with the original FTW and you are planning to
+> move to the time-based template you should check if you have done any
+> customization to these components.
+>
+> You should also note that `DateRangeFilter` is not in use in the time-based
+> template because the API does not support querying time-based availability.
+
+## Handling the timezones
+
+With time-based bookings timezones play a larger role than with night- and
```

Comment:
I think this should be "time zone".
https://en.wikipedia.org/wiki/Time_zone

Check all the other usages too (except when referring to moment-timezone lib)